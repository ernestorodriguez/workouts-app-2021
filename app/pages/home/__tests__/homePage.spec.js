/* eslint-disable react/jsx-props-no-spreading */
import React from 'react';
import { shallow } from 'enzyme';
import Page from '../index';

describe('Home page', () => {
  test.skip('should match snapshot', () => {
    const component = (
      <Page />
    );
    const wrapper = shallow(component);
    expect(wrapper).toMatchSnapshot();
  });
});
