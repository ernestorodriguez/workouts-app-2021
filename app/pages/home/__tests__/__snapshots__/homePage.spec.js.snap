// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page should match snapshot 1`] = `
<div
  className="home-page"
>
  <Header
    internal={false}
  />
  <h1
    className="home-page__title"
  >
    <small
      className="home-page__title--lighter"
    >
      Get Fit & Happy at Home
    </small>
    <br />
    WorkItUp!
  </h1>
  <Gallery />
</div>
`;
